Point to master branch for examples (#98)
yewstack/docs: src/getting-started/examples.md

# Learn through examples

The Yew repository is chock-full of [examples](https://github.com/yewstack/yew/tree/master/examples) \(in various states of maintenance\). We recommend perusing them to get a feel for how to use different framework features. We also welcome pull-requests and issues for when they inevitably get neglected and need some ♥️

> * [**Todo App** ](https://github.com/yewstack/yew/tree/master/examples/todomvc)

Reviewer: ```suggestion
* [**Todo App** ](https://github.com/yewstack/yew/tree/v0.16.0/examples/todomvc)
```

> * [**Custom Components**](https://github.com/yewstack/yew/tree/master/examples/custom_components)

Reviewer: ```suggestion
* [**Custom Components**](https://github.com/yewstack/yew/tree/v0.16.0/examples/custom_components)
```

> * [**Multi-threading \(Agents\)**](https://github.com/yewstack/yew/tree/master/examples/multi_thread)

Reviewer: ```suggestion
* [**Multi-threading \(Agents\)**](https://github.com/yewstack/yew/tree/v0.16.0/examples/multi_thread)
```

> * [**Timer Service**](https://github.com/yewstack/yew/tree/master/examples/timer)

Reviewer: ```suggestion
* [**Timer Service**](https://github.com/yewstack/yew/tree/v0.16.0/examples/timer)
```

> * [**Nested Components**](https://github.com/yewstack/yew/tree/master/examples/nested_list)

Reviewer: ```suggestion
* [**Nested Components**](https://github.com/yewstack/yew/tree/v0.16.0/examples/nested_list)
```

# Learn through examples

> The Yew repository is chock-full of [examples](https://github.com/yewstack/yew/tree/master/examples) \(in various states of maintenance\). We recommend perusing them to get a feel for how to use different framework features. We also welcome pull-requests and issues for when they inevitably get neglected and need some ♥️

Reviewer 2: ```suggestion
The Yew repository is chock-full of [examples](https://github.com/yewstack/yew/tree/v0.16.0/examples) \(in various states of maintenance\). We recommend perusing them to get a feel for how to use different framework features. We also welcome pull-requests and issues for when they inevitably get neglected and need some ♥️
```
